proc/community/src/main/java/org/neo4j/gds/triangle/TriangleCountStreamProc.java

```diff
@@ -65,13 +66,15 @@ public Stream<MemoryEstimateResult> estimate(
     protected Stream<Result> stream(ComputationResult<IntersectingTriangleCount, TriangleCountResult, TriangleCountStreamConfig> computationResult) {
         return runWithExceptionLogging("Graph streaming failed", () -> {
             var graph = computationResult.graph();
-            var result = computationResult.result();
+
+            var nodeProperties = nodeProperties(computationResult);
 
             return LongStream.range(0, graph.nodeCount())
-                .mapToObj(i -> new Result(
-                    graph.toOriginalNodeId(i),
-                    result.localTriangles().get(i)
-                ));
+                    .filter(i -> nodeProperties.value(i) != null)
+                    .mapToObj(i -> new Result(
+                            graph.toOriginalNodeId(i),
+                            ((LongValue)nodeProperties.value(i)).longValue()
```

Comment:
```suggestion
                            nodeProperties.longValue(i)
```

Reply:
Some context - the `value(i)` method returns value as the Neo4j internal datatype (internal ` org.neo4j.values.storable.Value`). This is mostly relevant for write but should not be relevant for stream